engine/engine/src/engine.cpp

```diff
@@ -852,6 +852,10 @@ namespace dmEngine
             return false;
         }
 
+        dmJobThread::JobThreadCreationParams job_thread_create_param;
+        job_thread_create_param.m_ThreadName = "DefoldJobThread1";
+        engine->m_JobThreadContext           = dmJobThread::Create(1, &job_thread_create_param);
```

Comment:
Right now we create a single worker thread, but this could be scaled up if we want. People recommend something like "number of cores + 1" or similar, but we have a bunch of other threads as well so I don't know what we want here.

Reply:
Yeah, threads are a resource.
E.g. Switch only has a few.
I'd say a game project config value should fit most projects.